apps/emqx/test/emqx_broker_helper_SUITE.erl

```diff
@@ -65,16 +58,134 @@ t_shard_seq(_) ->
     emqx_broker_helper:reclaim_seq(TestTopic),
```

Comment:
Both `{reclaim,create}_seq` seem to be only used here now.

Reply:
✂️